scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1060,6 +1060,10 @@ void BPF_STRUCT_OPS(layered_enqueue, struct task_struct *p, u64 enq_flags)
 	if (vtime_before(vtime, layer->vtime_now - layer_slice_ns))
 		vtime = layer->vtime_now - layer_slice_ns;
 
+	if (!tctx->all_cpus_allowed){
+		scx_bpf_dispatch(p, idx, slice_ns, enq_flags);
```

Comment:
also tracking idx is uninitialized, what I don't get is how this resolves stalls with the first config I tested.